scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -783,21 +790,224 @@ async def start_reborn_server(
+                diagnostics.trace_written = True
+            except Exception as trace_exc:
+                diagnostics.record(
+                    "diagnostic_error",
+                    source="trace_stop",
+                    error=repr(trace_exc),
+                )
             raise
         finally:
+            try:
+                diagnostics.write_summary()
+            except Exception as summary_exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)